src/app/users/users-profile/users-profile.component.ts

```diff
@@ -62,15 +63,15 @@ export class UsersProfileComponent implements OnInit {
     const updateDoc = Object.assign({ password: credentialData.password }, userDetail);
     this.changePasswordRequest(updateDoc).pipe(switchMap((response) => {
       if (response.ok === true) {
-        this.userDetail._rev = response._rev;
+        this.userDetail._rev = response.rev;
         return this.reinitSession(userDetail.name, credentialData.password);
       }
       return of({ ok: false, reason: 'Error changing password' });
     })).subscribe((res) => {
       if (res.ok === true) {
-        // TODO: Should notify user that password successfully changed or that there was an error
+        this.planetMessageService.showAlert('Password successfully changed');
       }
-    });
+    }, (error) => this.planetMessageService.showAlert(error.error.reason));
```

Comment:
Message should be readable for user. When I changed password for admin, it shows me function_clause . Please check it in the snapshot.
![screen shot 2018-03-22 at 3 56 41 pm](https://user-images.githubusercontent.com/7495421/37764450-a3c1263a-2db9-11e8-8ada-332ca380c776.png)